extensions/amp-story/1.0/test/validator-amp-story-amp-instagram.html

```diff
@@ -0,0 +1,48 @@
+<!--
+  Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  Tests for the amp-story-tooltip
```

Comment:
tooltip? also exists in other page.

please add a comment before the amp component in the document of whether it is `<!-- Valid -->` or `<!-- Invalid: reason -->`

Reply:
Woops, meant to set this as a draft PR. Didn't realize I clicked the wrong button. But thanks for the prompt review! I fixed your suggestions :)